docs/source/releases/v1_13_0a0.rst

```diff
@@ -1,16 +1,7 @@
 .. dropdown:: Distribution Statement
 
- | # # # Distribution Statement A. Approved for public release. Distribution is unlimited.
- | # # #
- | # # # Author:
- | # # # Naval Research Laboratory, Marine Meteorology Division
- | # # #
- | # # # This program is free software: you can redistribute it and/or modify it under
- | # # # the terms of the NRLMMD License included with this program. This program is
- | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
- | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
- | # # # for more details. If you did not receive the license, for more information see:
- | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
```

Comment:
NOTE: For some reason all of the contents of this release note had been removed.  I put the contents back - I don't know why the contents had been removed, but they seem to still be relevant (matplotlib_linear_norm -> linear_norm, and updating test scripts to geoips run data_fusion. Note I cahnged linear_norm BACK to matplotlib_linear_norm in 1.13.0)

Reply:
Sounds good! Thanks for doing that. I'm not sure how the release notes were removed either. This branch was created a while ago and that might have something to do with it.